src/course/config.json

```diff
@@ -12,7 +12,15 @@
         "_isEnabledOnTouchDevices": false,
         "_shouldSupportLegacyBrowsers": true,
         "_isTextProcessorEnabled": true,
-        "_isSkipNavigationEnabled": true
+        "_isSkipNavigationEnabled": true,
+        "_ariaLevels": {
```

Comment:
this will need adding to config.model.schema in both framework and in the authoring tool - and will need documenting somewhere too if not done already

Reply:
I've added a ticket for that work #1941